src/GitVersion.Core/VersionCalculation/BaseVersionCalculators/TaggedCommitVersionStrategy.cs

```diff
@@ -10,18 +11,19 @@ namespace GitVersion.VersionCalculation;
 /// </summary>
 public class TaggedCommitVersionStrategy : VersionStrategyBase
 {
-    private readonly IRepositoryStore repositoryStore;
+    private IRepositoryStore RepositoryStore { get; }
```

Comment:
Why are you changing `RepositoryStore` from `readonly` field to property? As it's still `private`, it can just remain a field, no? However, since `IRepositoryStore` seems to be in use in all `VersionStrategy` implementations, perhaps we can just pass it down into `VersionStrategyBase` and expose it as a `protected` property there?

Reply:
Good idea I'm doing this and move it to VersionStrategyBase.